### ISSUE TITLE
TH2 projections with live update

### ISSUE THREAD

**Issue by piberger** [root-project/jsroot#247]
Hi,

we have a TH2 with is updated once per second with redraw. When projection is enabled, there are glitches that the tooltip and the shaded area jump around when the mouse is moved while the plot is updated.
The shaded area often jumps between the position where the mouse pointer is and a random location in the plot.

As a workaround, we exported the `setPainterTooltipEnabled` function, set `JSROOT.settings.TooltipAnimation = 0;`, disable the tooltip before redraw and enable it again after.

    let obj = JSROOT.httpRequest(url, 'object').then(obj => {
        //disable tooltip
        JSROOT.setPainterTooltipEnabled(hist_painter, false);

        JSROOT.redraw("drawing", obj).then(painter => {
            //enable tooltip
            JSROOT.setPainterTooltipEnabled(hist_painter, true);
        });
    });

This works well, but we need to have a modified version of JSROOT code for it. Any better way or chance this can be fixed?
Thanks.

**Comment by linev** [comment]
Yes, I can reproduce glitch with drawing of histogram projection

And I fixed it in master and 7.2 branch - see https://github.com/root-project/jsroot/commit/6f030b3b44e5558528acea72059305d109963d3c

You could try it with `dev` version on https://jsroot.gsi.de

If necessary, I can backport it in older branches.

**Comment by piberger** [comment]
Thanks for the fix!
works with the dev branch :)
